lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_release_backmerge_pull_request_action.rb

```diff
@@ -110,17 +110,58 @@ def self.create_backmerge_pr(token:, repository:, title:, head_branch:, base_bra
 
         other_action.push_to_git_remote(tags: false)
 
-        pr_body = <<~BODY
-          Merging `#{head_branch}` into `#{base_branch}`.
+        # Live playground to edit this graph
+        # https://mermaid.live/edit#pako:eNqNkU1PwzAMhv9KZanqpWVFwCVHQOKCxoFrLl7itdGaZEoTTWjqf8eLOo2i8eGTPx77TewjKK8JBJTlUbqCzTgTRTEHJ6s6E18C7vtqkc4li8Y9BnSqX6MlBqoYkttV9Y_cW9AUGLxt2_Y7Nfb-8OStNfEVNzQwtcVhpCU1XUJ2p7KU7vzAXNlkmSLQQDjS6v7m4S7nVU9q51O8UsmSv7jzSEuho9Xc3pzaG-Oib_KXlxr_QL8onbuuVy9unvrXbKiBCV645qvme0mIPVmSINjVtMU0RAm8O0YxRf_-4RQIbqca0l5jpGeDXUALIu9_-gRejKzU
+        meramid_git_graph = <<~GRAPH
+          ```mermaid
+          %%{
+            init: {
+              'gitGraph': {
+                'mainBranchName': 'trunk',
+                'mainBranchOrder': 1000,
+                'showCommitLabel': false
+              }
+            }
+          }%%
+          gitGraph
+             branch #{head_branch}
+             checkout #{head_branch}
+             commit
+             commit
+             commit
+             branch #{intermediate_branch}
+             checkout #{intermediate_branch}
+             commit
```

Comment:
Technically, at this time, there is no commit on the integration branch. However, without the commit, I don't think the Mermaid graph doesn't convey the idea of the integration branch as well:

#### without `commit`

<img width="1683" alt="image" src="https://github.com/user-attachments/assets/bdd2a4ad-e21b-4028-ada0-15edc1aa911d">

#### with `commit`

<img width="1712" alt="image" src="https://github.com/user-attachments/assets/0fc7018c-7a30-413f-b73b-c8e9787ca4aa">

Reply:
Yep, that was also a limitation I remember seeing when I tried it a while ago. It I think it makes sense and is ok to have that commit visually added there anyway.

> Technically, at this time, there is no commit on the integration branch

Besides, this is not entirely true. For example, on Tumblr there is a commit done on the intermediate branch after it is cut from `release/`—the goal of that extra commit being to `git checkout $basebranch -- version.xcconfig` to auto-resolve the conflict that would otherwise be present on that file due to it evolving separately on `release/*` branch (for daily betas) vs `develop` (for daily alphas).

This is also why this Fastlane action has a `ConfigItem` allowing you to pass a `Proc` to add arbitrary commits on the intermediate branch after it's cut but before the PR is created, btw 😉

Reply:
It also feels strange that the commits on `trunk` seem to have happened only after the commits from `release/4.53` in the screenshot above; It's like if the 3 commits from `trunk` that were illustrated in this Mermaid graph were only pushed after `release/*` and `merge/release-*-into-*` had been created, instead of suggesting that the `trunk` branch has been existing and living its life for way longer before `release/*` branch… 🤔